source/test/sgp_mode_test/SGPDiversity.test.cc

```diff
@@ -105,7 +104,9 @@ TEST_CASE("AlphaDiversity", "[sgp]") {
     
     double test_value = AlphaDiversity(test_vec);
 
-    THEN("") { REQUIRE(test_value == expected_value); }
+    THEN("Calculation is correct") { 
```

Comment:
Instead of just 'calculation is correct', you should state what it should actually be: "alpha diversity is equal to 1"

Reply:
This still needs to be fixed